Unix/wsman/wsmanparser.c

```diff
@@ -3241,29 +3240,66 @@ int WS_ParseFaultBody(
                 }
                 else if (Tcscmp(e.data.data, ZT("WSManFault")) == 0)
                 {
+                    /* fault->mi_result should be MI_RESULT_FAILED for WSManFault */
+                    fault->mi_result = MI_RESULT_FAILED;
```

Comment:
OK, so I did not find a mapping when I thought there was. I thought it was done in wsmanerrorhandling.c where it does most of the error processing.
For now lets leave it the way you did it.